emit 'message' event on message

nicksellen opened the pull request:
I know you have wildcards on the [TODO](https://github.com/easternbloc/node-stomp-client/blob/master/TODO.md) list however all I need is a simple catch-all subscription which can be achieved just by emitting a 'message' event on each message.

Also notice that wildcard subscriptions come in different flavours - Apollo uses `**` for nested wildcard matching, whereas ActiveMQ uses `>` - I'm not sure it would be such a good idea to try and reimplement the wildcard logic yourself for this reason.

If you're happy with the idea, I can add some tests and docs too.

easternbloc commented:
Yeah as a simple catch all I think that could be OK. Could you add tests and update the docs in the same PR?

nicksellen commented:
I added docs/tests :)

I also updated the devDependencies as nodeunit 0.7.4 uses some deprecated node stuff:

```
/data/code/node-stomp-client/node_modules/nodeunit/lib/utils.js:14
    Script = process.binding('evals').Script || process.binding('evals').NodeS
                     ^
Error: No such module: evals
```

easternbloc commented:
@nicksellen available on npm as version 0.8.0 enjoy

nicksellen commented:
super, thanks :)
